services/madoc-ts/src/routes/iiif/manifests/get-manifest.ts

```diff
@@ -46,7 +46,8 @@ export const getManifest: RouteMiddleware<{ id: string }> = async context => {
 
   const manifest = table.manifests[`${manifestId}`] || {
     id: manifestId,
-    label: { none: ['Untitled manifest'] },
+    label: { none: ['Manifest not found'] },
+    source: 'not-found',
```

Comment:
I think this is a better way of handling missing manifests, as we can detect it. If we have the error handling in the loaders, we might be able to throw an error here instead - although I'm not sure if that will cause problems elsewhere.